docs/reference/anitya.lib.backends.rst

```diff
@@ -0,0 +1,12 @@
+Anitya Backends
+===============
+
+These are the backends supported by Anitya
+
+Crates.io
+---------
+
+.. automodule:: anitya.lib.backends.crates
```

Comment:
The docs project is definitely a work-in-progress (there's a bunch of build warnings/errors from some docblocks). My plan is to chip away at the docs and add the modules as they're completed. The ones with docblocks don't have module-level documentation, so it makes the rendered results look a little odd.